using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md

```diff
@@ -136,6 +136,115 @@ ACPCore.trackState("state name", data: ["key": "value"])
 {% endtab %}
 {% endtabs %}
 
+
+
+## Collect launch information
+
+You can provide the user info to the SDK from various launch points in your application. 
+
+{% hint style="info" %}
+
+If **Analytics** extension is enabled in your SDK, the collection of this launch data will result in an Analytics request being sent. Other extensions in the SDK might use the collected data, for example, as a rule condition for an In-App Message.
+{% endhint %}
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Java
+
+In Android, you can collect data from the Activity or the context to be used later by the SDK.
+
+The `collectLaunchInfo` API marshals the Activity instance and extracts the intent data or extras. It should be called to support the following use cases:
+
+- Tracking Deep Link click-throughs
+  - Update `AndroidManifest.xml` to support intent-filter in the activity with the intended action and type of data.
```

Comment:
please add "the" between "support" and "intent" and remove the hyphen between "intent" and filter"